### ISSUE TITLE
SnomedCT <-> kmehr

### ISSUE THREAD

**Issue by hansdekeersmaecker** [hl7-be/vaccination#124]
In the care set for BeVaccination SNOMEDCT codes are used for the VaccineCode https://www.ehealth.fgov.be/standards/fhir/vaccination/ValueSet-be-vs-vaccine-code.html. 
But we currently have to support in different exports (sumehr, PMF...) kmehr indications of the current kmehr model: https://www.ehealth.fgov.be/standards/kmehr/en/tables/vaccine-indication-codes.

Because we need to support both systems at the same time, can a mapping table be provided so we are sure to use the correct mapping between SNOMEDCT & kmehr codes.

**Comment by bdc-ehealth** [comment]
This looks like terminology work to me... The impact on the FHIR structures is low/non-existant. We need the input of a medically trained person to do this.

**Comment by costateixeira** [comment]
To be discussed in the WG.

**Comment by hansdekeersmaecker** [comment]
This will probably have no impact on fhir structure indeed, but is very important for the implementation because we need to foresee both systems in parallel. For me it's indeed a medically trained person that should have look at this.

**Comment by costateixeira** [comment]
Yes, hence to discuss in the wg - we cannot dismiss the possible impact before discussing e.g. mapping tables.
And i guess you mean a clinical terminologist? 
The wg should clarify what is needed from the terminologist as per Hans's request.

**Comment by KarlienHL7Belgium** [comment]
This has been raised before. The WG thinks that it should be part of the eHealth platform publication
The work should be led by RIZIV, by @anne nerenhausen, Karlien will contact her and point out that it is critical

**Comment by KarlienHL7Belgium** [comment]
RIZIV confirms that the mapping table will be available by end of June, including publication by eHealth platform
